internal/plugin/service/organization/organization_user_group_member.go

```diff
@@ -0,0 +1,247 @@
+func NewOrganizationUserGroupMembersResource() resource.Resource {
+	return &organizationUserGroupMembersResource{}
+}
+
+// organizationUserGroupMembersResource is the organization resource implementation.
+type organizationUserGroupMembersResource struct {
+	// client is the instance of the Aiven client to use.
+	client *aiven.Client
+
+	// typeName is the name of the resource type.
+	typeName string
+}
+
+// organizationUserGroupMembersResourceModel is the model for the organization resource.
+type organizationUserGroupMembersResourceModel struct {
+	// ID is the identifier of the organization.
+	OrganizationID types.String `tfsdk:"organization_id"`
+
+	// ID is the identifier of the organization user group.
+	OrganizationGroupID types.String `tfsdk:"group_id"`
+
+	// ID is the identifier of the organization user group member.
+	OrganizationUserID types.String `tfsdk:"user_id"`
+
+	// Last activity time of the user group member.
+	LastActivityTime types.String `tfsdk:"last_activity_time"`
+
+	// Timeouts is the configuration for resource-specific timeouts.
+	Timeouts timeouts.Value `tfsdk:"timeouts"`
+}
+
+// Metadata returns the metadata for the organization resource.
+func (r *organizationUserGroupMembersResource) Metadata(
+	_ context.Context,
+	req resource.MetadataRequest,
+	resp *resource.MetadataResponse,
+) {
+	resp.TypeName = req.ProviderTypeName + "_organization_user_group_member"
+
+	r.typeName = resp.TypeName
+}
+
+func (r *organizationUserGroupMembersResource) TypeName() string {
+	return r.typeName
+}
+
+// Schema returns the schema for the organization resource.
+func (r *organizationUserGroupMembersResource) Schema(
+	ctx context.Context,
+	_ resource.SchemaRequest,
+	resp *resource.SchemaResponse,
+) {
+	resp.Schema = util.GeneralizeSchema(ctx, schema.Schema{
+		Description: "Creates and manages an organization user group members in Aiven.",
+		Attributes: map[string]schema.Attribute{
+			"organization_id": schema.StringAttribute{
+				Description: "Identifier of the organization.",
+				Required:    true,
+			},
+			"user_id": schema.StringAttribute{
+				Description: "Identifier of the organization user group member.",
+				Required:    true,
+			},
+			"group_id": schema.StringAttribute{
+				Description: "Identifier of the organization user group.",
+				Required:    true,
+			},
+			"last_activity_time": schema.StringAttribute{
+				Description: "Last activity time of the user group member.",
+				Computed:    true,
+			},
+		},
+	})
+}
+
+// Configure configures the organization resource.
+func (r *organizationUserGroupMembersResource) Configure(
+	_ context.Context,
+	req resource.ConfigureRequest,
+	resp *resource.ConfigureResponse,
+) {
+	if req.ProviderData == nil {
+		return
+	}
+
+	client, ok := req.ProviderData.(*aiven.Client)
+	if !ok {
+		resp.Diagnostics = util.DiagErrorUnexpectedProviderDataType(resp.Diagnostics, req.ProviderData)
+
+		return
+	}
+
+	r.client = client
```

Comment:
I think this might be casted in an unsafe way. I won't compile otherwise.

Reply:
I'm reusing the same code that we have for organization resource